packages/transformers/sass/src/SassTransformer.js

```diff
@@ -1,93 +1,79 @@
 // @flow
-import type {FilePath} from '@parcel/types';
-import type {FileSystem} from '@parcel/fs';
-import type {PluginLogger} from '@parcel/logger';
-
 import {Transformer} from '@parcel/plugin';
-import {promisify, resolve} from '@parcel/utils';
-import {dirname, join as joinPath} from 'path';
+import {promisify} from '@parcel/utils';
+import path from 'path';
 import {EOL} from 'os';
 import SourceMap from '@parcel/source-map';
 
 // E.g: ~library/file.sass
 const WEBPACK_ALIAS_RE = /^~[^/]/;
 
-let didWarnAboutNodeSass = false;
-async function warnAboutNodeSassBeingUnsupported(
-  fs: FileSystem,
-  filePath: FilePath,
-  logger: PluginLogger,
-) {
-  if (!didWarnAboutNodeSass) {
-    try {
-      await resolve(fs, 'node-sass', {basedir: dirname(filePath)});
-      logger.warn({
-        origin: '@parcel/transformer-sass',
-        message:
-          '`node-sass` is unsupported in Parcel 2, it will use Dart Sass a.k.a. `sass`',
-      });
-    } catch (err) {
-      if (err.code !== 'MODULE_NOT_FOUND') {
-        throw err;
-      }
-    } finally {
-      didWarnAboutNodeSass = true;
-    }
-  }
-}
-
 export default new Transformer({
-  async getConfig({asset, resolve, options}) {
-    let config = await asset.getConfig(['.sassrc', '.sassrc.js'], {
+  async loadConfig({config, options}) {
+    let configFile = await config.getConfig(['.sassrc', '.sassrc.js'], {
       packageKey: 'sass',
```

Comment:
Same with packageKey. Are these supported by other tools or did we invent them? It's pretty inconsistent in parcel when we support them and when not...